.github/workflows/refapp-2x-phoneNumberValidator.yml

```diff
@@ -0,0 +1,41 @@
+name: RefApp 2.x Phone Number validator
```

Comment:
What is your thought about `Phone Number Validator`  vs `Phone Number Validation`?